packages/babel-traverse/src/index.js

```diff
@@ -9,7 +9,8 @@ import * as cache from "./cache";
 
 export { default as NodePath } from "./path";
 export { default as Scope } from "./scope";
-export { default as Hub } from "./hub";
+import type Hub from "./hub";
+export type { Hub };
```

Comment:
no it isn't, this is how flow export types. this has no influence to the compiled code.